### ISSUE TITLE
nested comma separated selectors bug?

### ISSUE THREAD

**Issue by russelgal** [cristianbote/goober#182]
```
const Div = css`
  color: red;
  p,  b, i {
    display: block;
  }
`;
```
produced
`.go1684683309{color:red;}.go1684683309 p,b,i{display:block;}`

https://codesandbox.io/s/solid-styled-components-forked-jg3b5?file=/index.js:111-184

#49

**Comment by cristianbote** [comment]
Hey @russelgal, thanks for opening this issue. Indeed 🤔 something is not working right. We had this issue before and I have a feeling we fixed it. Let me search for it. I'll get back with it.

**Comment by cristianbote** [comment]
Alrighty here it is https://github.com/cristianbote/goober/pull/172/files#diff-9c7fd86f60e4a5921e586a27bafc1bf3R19

That should take care of the nested selector, but something is not working as it should. Looking into it.

**Comment by cristianbote** [comment]
Alrighty, look over this particular issue and it boils down to how `parse` is balancing the `selector` and the current `key`. I'm looking for a correct fix to handle both of these.

**Comment by cristianbote** [comment]
The fix is published in `goober@2.0.7`. Let me know if you run into more issues! This was an excellent issue! 👌 Thank you!

**Comment by russelgal** [comment]
thanks for the super fast fix! 👍 and ... for the kind words 😉
keep up the good work!
